cpmpy/transformations/linearize.py

```diff
@@ -69,16 +69,19 @@ def linearize_constraint(lst_of_expr, supported={"sum","wsum"}, reified=False):
 
         # boolvar
         if isinstance(cpm_expr, _BoolVarImpl):
-            newlist.append(sum([cpm_expr]) >= 1)
+            if "bv" in supported:
```

Comment:
Hmm we typically don't do this as the supported set contains names of constraints, but I can see why it's done here. 
Personally, I would prefer it to stay a sum but no strong objection to this change.
In any case, update the description of the `linearize` normal form above.